entity_api/src/organization.rs

```diff
@@ -206,4 +218,23 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn find_by_external_id_queries_by_external_id() -> Result<(), Error> {
+        let external_id = Uuid::new_v4();
+        let db = MockDatabase::new(DatabaseBackend::Postgres).into_connection();
+
+        let _ = find_by_external_id(&db, &external_id.to_string()).await;
+
+        assert_eq!(
+            db.into_transaction_log(),
+            [Transaction::from_sql_and_values(
+                DatabaseBackend::Postgres,
+                r#"SELECT "organizations"."id", "organizations"."external_id", "organizations"."name", "organizations"."logo", "organizations"."created_at", "organizations"."updated_at" FROM "refactor_platform"."organizations" WHERE "organizations"."external_id" = $1 LIMIT $2"#,
+                [external_id.into(), 1u64.into()]
```

Comment:
`1u64` is interesting to compare `external_id` to. Is this right, I would expect it to be a `String`?

Reply:
Good question! This actually is comparing `1u64` to the `$2` value in `LIMIT $2` which is added by the `.one()` function in

```Rust
    let organization = Entity::find()
        .filter(Column::ExternalId.eq(uuid))
        .one(db)
        .await?;
```
the `into()` call converts the `1u64` into a [BigUnsigned](https://docs.rs/sea-orm/latest/sea_orm/enum.Value.html#variant.BigUnsigned) which is, I believe, how seaORM represents the underlying postgres type

Reply:
That makes sense, thanks for explaining it...it wasn't super obvious from what it looks like at first but now it does.